packages/cli/scripts/preinstall.js

```diff
@@ -24,11 +24,11 @@ function isPackageInstalledGlobally(name) {
 }
 
 function prismaIsInstalledGlobally() {
-  const prisma2InstalledGlobally = isPackageInstalledGlobally('prisma2')
-  if (prisma2InstalledGlobally) {
+  const prismaInstalledGlobally = isPackageInstalledGlobally('prisma')
+  if (prismaInstalledGlobally) {
     return {
-      ...prisma2InstalledGlobally,
-      name: 'prisma2',
+      ...prismaInstalledGlobally,
+      name: 'prisma',
     }
   }
```

Comment:
This logic seems to have relied on the deprecated bin/executable, and can not any more.

Reply:
Looks to me that the whole `prismaIsInstalledGlobally()` function and usage could go away as later in this file it's only used to check if `prisma2` is installed globally and error with a message (see line 62+)

Reply:
Oh you are right, my change breaks that functionality.

I had considered removing that bit as well, but it is not about the `prisma2` bin of the current Npm package, but about a `prisma2` npm package we used to have. With us not having the`prisma2` bin any more, we honestly don't care if someone still has that old Npm package installed I think.

Reply:
This feels pretty drastic, but along those lines?
https://github.com/prisma/prisma/pull/19439

Reply:
Yes that's it, I think this removal is due and the clean slate very nice

Reply:
Merged and also removed dependency.